sys/net/include/etx_beaconing.h

```diff
@@ -26,43 +25,70 @@
 extern "C" {
 #endif
 
-//[option|length|ipaddr.|packetcount] with up to 15 ipaddr|packetcount pairs
-// 1 Byte 1 Byte  1 Byte  1 Byte
+/**
+ * @brief [option|length|ipaddr|packetcount] with up to 15 ipaddr|packetcount pairs
+ *        | option | length | ipaddr  | packetcount |
+ *        | ----:  | :----: | :----:  | :----:      |
+ *        | 1 Byte | 1 Byte | 1 Byte  | 1 Byte      |
+ */
 #define ETX_BUF_SIZE   (32)
-
 #define ETX_RCV_QUEUE_SIZE     (128)
 
-/*
- * Default 40, should be enough to get all messages for neighbors.
- * In my tests, the maximum count of neighbors was around 30-something
+/**
+ * @brief defining the number of neighbours
  */
 #if ENABLE_DEBUG
-#define ETX_MAX_CANDIDATE_NEIGHBORS (15) //Stacksizes are huge in debug mode, so memory is rare
+ /**Defines the maximum number of neighbours in debug mode, we fix it at 15
+  */
+#define ETX_MAX_CANDIDATE_NEIGHBORS (15)
+/**Defines the maximum number of neighbours in normal mode default is 40
```

Comment:
Change this to a normal comment, i.e. `/* asdf */` instead of `/** asdf */`, put the closing comment slash on the same line, don't forget to add a space after the opening `/*`